scout/server/blueprints/cases/controllers.py

```diff
@@ -1424,13 +1424,13 @@ def _matching_causatives(
     other_causatives_in_default_panels = []
 
     for causative in matching_causatives:
-        hgnc_ids = [gene.get("hgnc_id") for gene in causative.get("genes", [])]
+        hgnc_ids = {gene.get("hgnc_id") for gene in causative.get("genes", [])}
```

Comment:
This is only ever used as a set anyway; no need to call the set constructor several times.